pkg/util/pointers/pointers.go

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pointers
+
+import "time"
+
+// Bool returns a boolean pointer for the given value.
+func Bool(v bool) *bool { return &v }
```

Comment:
Let's please not do this

Reply:
If this is a one-time thing you could just oneline this. E.g. https://github.com/TheThingsNetwork/lorawan-stack/blob/9be9736f91cc0ea954e7773d675c738dc87ce4fd/cmd/internal/shared/networkserver/config.go#L36

Otherwise just have a non-exported `boolPtr` next to the usage.

Reply:
Why not? Then I need to define the same `boolPtr ` and functions for each type in 4 different files.

Reply:
Yes, that is inherent to Go